### DISCUSSION TITLE
Moving / rearranging the tokens in the plugin

### DISCUSSION THREAD

**Discussion by Bartvlaemynck** [six7/figma-tokens#496] [Q&A]
Is there a way at the moment to move/rearrange the tokens in the plugin? Now they are in creation order I think.

**Comment by robintweeter** [comment]
Would love a fix for this, renaming so it's alphabetical or numeric would work of allowing drag and drop within the token elements would work as well. Trying to get a Dark Mode and Light Mode to have the same order is not easy to achieve when any edits can mess up the existing order.
